priceloop_api/priceloop/test/test_table.py

```diff
@@ -0,0 +1,145 @@
+import os
+
+import pandas as pd
+import pytest
+
+from priceloop_api.priceloop.extras import (
+    Column,
+    ColumnMixedTypes,
+    ColumnType,
+    PatchMode,
+    Table,
+    TableUtils,
+    get_priceloop_client,
+)
+
+
+@pytest.fixture
+def sample_table() -> Table:
+    return Table(
+        os.environ["NOCODE_WORKSPACE"],
+        "table1",
+        [
+            Column("name", ColumnType.Text),
+            Column("age", ColumnType.Number),
+            Column("date_of_birth", ColumnType.Date),
+            Column("gpa", ColumnType.Number),
+            Column("json_field", ColumnType.Json),
+        ],
+    )
+
+
+@pytest.fixture
+def sample_dataframe1():
+    df = pd.DataFrame(
+        data={
+            "name": ["John", "Alice", "Bob"],
+            "age": [25, 30, 22],
+            "date_of_birth": pd.to_datetime(["1997-05-15", "1992-08-22", "2000-01-10"]),
+            "gpa": [3.3, 2.9, 3.0],
+            "json_field": ["{}", '{"a":"object"}', '{"1":3}'],
+        },
+    )
+    return df
+
+
+@pytest.fixture
+def sample_dataframe2():
+    return pd.DataFrame(
+        data={
+            "name": ["John", 22, "Bob"],
+            "age": [25, 30, 22],
+            "date_of_birth": pd.to_datetime(["1997-05-15", "1992-08-22", "2000-01-10"]),
+            "gpa": [3.3, 2.9, 3.0],
+            "json_field": ["{}", '{"a":"object"}', '{"1":3}'],
+        },
+    )
+
+
+class TestTable:
+    def test_set_df_success(self, sample_table, sample_dataframe1):
+        sample_table.set_df(sample_dataframe1)
+
+    def test_set_df_fail(self, sample_table, sample_dataframe2):
+        assert True
+        try:
+            sample_table.set_df(sample_dataframe2)
+            assert False
+        except ColumnMixedTypes:
+            assert True
+
+    def test_from_df_constructor(self, sample_dataframe1):
+        Table.from_df(sample_dataframe1, os.environ["NOCODE_WORKSPACE"], "test 3")
+        assert True
+
+
+class TestTableUtils:
+    def test_initiate_table_on_platform(self, sample_table):
+        client = get_priceloop_client()
+        assert sample_table.is_table_on_platform(client) == False
+        assert sample_table.initialize_table(client)
+
+    def test_get_table_data(self, sample_table):
+        client = get_priceloop_client()
+        TableUtils.read_table(
+            sample_table.workspace,
+            sample_table.name,
+            client,
+        )
+        assert True
```

Comment:
what is the purpose of all the `assert True` here. If the function fail, the test fails as well.